Sources/XCHammer/main.swift

```diff
@@ -88,7 +92,8 @@ struct GenerateOptions: OptionsProtocol {
 
     static func evaluate(_ m: CommandMode) -> Result<GenerateOptions, CommandantError<ClientError>> {
         return create
-            <*> m <| Argument(usage: "Path to the XCHammerConfig yaml file")
+            <*> m <| Argument(defaultValue: nil,
+                 usage: "Path to the XCHammerConfig yaml file")
```

Comment:
Consider adding a note that it reads in `XCHammer.yaml` by default.

Reply:
good call